README.md

```diff
@@ -75,6 +76,7 @@ custom:
       migrate: true
       seed: true
       convertEmptyValues: true
+      userStop: true
```

Comment:
Here if we keep userStop: true, it could potentially affect the behavior of existing users of the library. Shall we make it false by default. What do you think?